eslint.config.js

```diff
@@ -0,0 +1,61 @@
+import js from '@eslint/js';
+import prettierPlugin from 'eslint-plugin-prettier';
+import simpleImportSortPlugin from 'eslint-plugin-simple-import-sort';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+	// register all of the plugins up-front
+	{
+		plugins: {
+			['@typescript-eslint']: tseslint.plugin,
+			['simple-import-sort']: simpleImportSortPlugin,
+			['prettier']: prettierPlugin,
+		},
+	},
+
+	// config with just ignores is the replacement for `.eslintignore`
+	{
+		ignores: ['**/dist', '**/package-lock.json', '**/package.json'],
+	},
+
+	// extends ...
+	js.configs.recommended,
+	...tseslint.configs.recommended,
+
+	// base config
+	{
+		rules: {
+			'@typescript-eslint/no-explicit-any': 'off',
```

Comment:
What is the justification for disabling explicit any? There are bascially no cases where we want to use `any`. The suggestions provided in the rule documentation for `any` alternatives are pretty strong:

https://typescript-eslint.io/rules/no-explicit-any/

Generally, instead of any we would want to provide the minimum data type expected, or use `unknown` so any handling of that data will validate the contents.

Reply:
We're using `any` mostly on Request handlers and catching errors for simplicity. Will remove this exception rule and fix to the corresponding types